google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -41,16 +41,17 @@ class Pipeline(object):
 from airflow.contrib.operators.bigquery_table_delete_operator import BigQueryTableDeleteOperator
 from airflow.contrib.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
 from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
-from datetime import datetime, timedelta
+from datetime import timedelta
+from pytz import timezone
 
 """
 
   _default_args_format = """
     'owner': 'Datalab',
     'depends_on_past': False,
     'email': ['{0}'],
-    'start_date': datetime.strptime('{1}', '{3}'),
-    'end_date': datetime.strptime('{2}', '{3}'),
+    'start_date': {1},
```

Comment:
These are actually part of the airflow python spec, and cannot be changed. I plan to make it clear in our documentation in Datalab that the allowed timezone is only UTC.